src/main/features/linux/mediaKeysDBus.js

```diff
@@ -19,6 +19,21 @@ try {
       iface.GrabMediaPlayerKeys(0, 'org.gnome.SettingsDaemon.MediaKeys'); // eslint-disable-line
     }
   });
+  session.getInterface('org.mate.SettingsDaemon', '/org/mate/SettingsDaemon/MediaKeys',
```

Comment:
This is exact code duplication with two strings different.  Can you extract it to another function and pass in a single parameter (`gnome` and `mate`)